src/components/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,56 @@
+import { ButtonBase } from '@mui/material';
+import React from 'react';
+import styled from 'styled-components';
+
+const MainContent = () => {
+	const MainContainer = styled.div`
+		height: 100vh;
+		width: 1550px;
+		border-color: black;
+		border-radius: 10px;
+		float: center;
+		position: absolute;
+		left:310px;
+		top:100px;
+		
+		background-image: url('https://img.freepik.com/free-photo/young-woman-summer-clothes-listening-music-wearing-headphones-singing-along-favorite-song-dancing-earphones-standing-against-blue-background_1258-122074.jpg?w=1800&t=st=1676928567~exp=1676929167~hmac=612d6f0c71d60a4eed6a9a20e79296aa8e4eae1f64d948e141910009e5faa968');
```

Comment:
I think it would be better to extract this url to a constant - it's too large. Create folder consts in src and export it to a e.g. imageUrls.ts